source/grid/grid_generator.cc

```diff
@@ -3449,20 +3449,23 @@ namespace GridGenerator
 
     std::vector<Point<3> > points(n_slices*input.n_vertices());
     std::vector<CellData<3> > cells;
-    cells.reserve((n_slices-1)*input.n_active_cells());
+    cells.reserve(n_slices*input.n_active_cells());
```

Comment:
I don't think this is correct. If we have ```n_slices``` vertices in the extruded direction, we will get ```n_slices-1``` cells in that direction. Notice that the cell loop below loops while ```slice<n_slices-1```.